src/Models/Thread.php

```diff
@@ -195,6 +206,24 @@ public function scopeForUserWithNewMessages(Builder $query, $userId)
             ->select($threadsTable . '.*');
     }
 
+    /**
+     * Returns threads between given user ids.
+     *
+     * @param \Illuminate\Database\Eloquent\Builder $query
+     * @param array                                 $participants
+     *
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function scopeBetweenOnly(Builder $query, array $participants): Builder
```

Comment:
Can you add some tests around this method? The PR doesn't seem to include any. Other than that, it's looking good

Reply:
I've already wrote one, mostly it got lost when I fixed the conflict, I'll do it now.